### DISCUSSION TITLE
Mistake while explaining the concept of duet Video 12 Federated Learning

### DISCUSSION THREAD

**Discussion by AVI18794** [OpenMined/courses#274] [Feedback] [answered]
In video 12  at 3:41 while explaining the duet variable on the data scientist side the duet will be the pointer to the data owner's server . There is a slight mistake in the video where instead of data scientist  side it was by mistake pronounced as data owner side twice.

**Comment by znreza** [comment, marked as answer]
Hi @AVI18794 yes that was a slight mistake in the audio! It is already mentioned on the slide at the top that it should be data scientist instead. Thanks for your concern.

**Reply by yemikifouly** [reply]
Thanks @znreza! 🙌 
@AVI18794 Thank you for pointing this out Avi! 🙌 In addition to what Zarreen said about the issue not appearing on the slide, I will try to have this added as erratum in the notes below the video.
